packages/flutter/lib/src/widgets/feedback.dart

```diff
@@ -127,8 +127,9 @@ abstract final class Feedback {
 
   /// Provides platform-specific feedback for a long press.
   ///
-  /// On Android the platform-typical vibration is triggered. On iOS this is a
-  /// no-op as that platform usually doesn't provide feedback for long presses.
+  /// On Android the platform-typical vibration is triggered. On iOS a
+  /// heavy-impact haptic feedback is triggered, alongside the click system
+  /// sound.
```

Comment:
Nit: "sound, which was observed to be the default behavior on a physical `<iPhone version>` running `<iOS version>`."